pyiron/atomistics/structure/atoms.py

```diff
@@ -1710,6 +1741,32 @@ def plot3d(
 
         view.camera = camera
         view.background = background
+
+        if camera_axis != "x" and camera_axis != "y" and camera_axis != "z":
+            warnings.warn(
+                "Choose between x, y, or z"
+            )
+
+        zoom_out = 14  # higher = farther away
```

Comment:
Should not this be an argument? I could see it being useful to set the zoom from the function even when no rotation is applied.

Reply:
I will put in a `camera_zoom` flag which defaults to `None`, and then sets it to 14, if no other value is specified. I think users will be pretty confused to see this random number 14 as a default.